site/content/entry/yuzu-progress-report-dec-2021/index.md

```diff
@@ -0,0 +1,371 @@
+In both cases those drivers are most likely years old (yuzu can run on AMD GPUs from 2012) and either lack Vulkan support at all, or only support a portion of what’s 
+needed to run yuzu.
+Also, since laptops by default connect the display directly to the integrated GPU, that’s the first Vulkan driver that will be seen, so it’s critical to have the latest 
+GPU driver installed *even if* your laptop has a dedicated NVIDIA GPU running the latest driver.
+
+While telling AMD users to [manually download and install updated drivers](https://www.amd.com/en/support) is a viable option and works as it should, in its 
+*infinite wisdom*, Intel decided to block manual installation of its own official drivers if a custom laptop vendor driver is in use (those modified drivers are usually 
+created to cheat on battery life metrics and/or to save money on cooling).
+
+The only alternative in those cases is to [manually download the ZIP version](https://www.intel.com/content/www/us/en/download/19344/intel-graphics-windows-dch-drivers.html) 
+of the driver, unpack it, go to Device Manager, right-click the correct GPU in Display Adapters, select `Update Driver Software…`, `Browse my computer for driver software`, 
+`Let me pick from a list of device drivers on my computer`, `Have Disk…`, then finally browse to the folder where the driver was unpacked and select the `iigd_dch.inf` file.
+What a very intuitive and user-friendly way to update a GPU driver, great job Intel…
+
+Here's a [video tutorial](https://www.youtube.com/watch?v=BZG50Nm5sOM&t=72s) for those that prefer visual aid over our rambling. Just make sure to use the `iigd_dch.inf` 
+file instead of the one shown in the slightly outdated video.
+Other mentioned optimizations on the video no longer apply.
+
+With this *easy job* done, the Intel GPU gets full Vulkan support, runs at its intended performance, and has access to all the new features, fixes, and performance 
+improvements that the driver developers worked on. 
+The driver is also allowed to auto-update on new official releases.
+
+Known software that uses broken Vulkan injectors are outdated screen recorders like Bandicam, Action!, and even OBS.
+We strongly recommend using an up to date OBS, the native encoders from the GPU vendor (Radeon ReLive and Geforce Experience), or the integrated Xbox Game Bar on Windows.
+Overwolf and GShade are also known to break Vulkan support, so we **strongly** recommend avoiding them.
+
+## Future projects
+
+`Project Gaia`, is progressing smoothly. As a tip, SSD users will notice improvements once it is released.
+
+Blinkhawk informs us that `Project Y.F.C.` will be grouping together the changes into smaller pieces, to get more progressive updates instead of delaying for a big release
+that would require more testing time.
+We continue to plan to add several of GPU features that have been pending, for example:
```

Comment:
```suggestion
Blinkhawk informs us that `Project Y.F.C.` will be released in smaller chunks in order to push more progressive updates instead of delaying for a big release
that would require more testing time. We want to get these updates in your hands as soon as possible!
We continue to plan to add several GPU features that have been pending. Here's a screenshot as an example:
```